### PR TITLE
ConceptProcessing.bestReactionForGoal()

### PR DESCRIPTION
**Pull request by PtrMan**
* split static method into two
* condtion inversion

### CODE REVIEW

File: src/main/java/org/opennars/control/ConceptProcessing.java [hunk 1]
```diff
@@ -439,107 +439,128 @@ else if(task.isInput() && !quesTask.getTerm().hasVarQuery() && quesTask.getBestS
...
+                    return;
+                }
+
+                concept.memory.decisionBlock = concept.memory.time() + Parameters.AUTOMATIC_DECISION_USUAL_DECISION_BLOCK_CYCLES;
+                generatePotentialNegConfirmation(nal, meta.executable_precond.sentence, meta.executable_precond.budget, meta.mintime, meta.maxtime, 2);
+            }
+        }
+    }
+
+    private static BestOpWithMetainfo calcBestOpWithMeta(final DerivationContext nal, final Concept concept, final Sentence projectedGoal) {
```
[the remark this answers is not in the document]
Comment (PtrMan) [reply src/main/java/org/opennars/control/ConceptProcessing.java]: This is not my problem :D

**Comment by PtrMan** [comment]
@patham9 are the names fine now?

**Comment by patham9** [comment]
approved, resolve the conflicts please. Additionally, think about whether bestReactionForGoal is still necessary given that it only consists of two lines of code and is just called once. What do you think?

**Comment by PtrMan** [comment]
I will rework this

**Comment by patham9** [comment]
since this one got stuck: https://github.com/opennars/opennars/pull/307

**Pull request closed without merging**